src/components/overflow-menu/overflow-menu.ts

```diff
@@ -64,21 +64,22 @@ class BXOverflowMenu extends HostListenerMixin(LitElement) implements BXFloating
     super.connectedCallback();
   }
 
-  attributeChangedCallback(name, old, current) {
-    if (old !== current) {
-      if (name === 'open' && !this._menuBody) {
+  createRenderRoot() {
+    return this.attachShadow({ mode: 'open', delegatesFocus: true });
+  }
+
+  updated(changedProperties) {
```

Comment:
This change was originally introduced for Angular which puts property binding the precedence, and thus tracking change in property rather than attribute.